README.md

```diff
@@ -1,28 +1,26 @@
-
-[![Download@3.2](https://img.shields.io/github/downloads/martinet101/ElevenClock/3.2.0/total?style=for-the-badge)](https://github.com/martinet101/ElevenClock/releases/latest/download/ElevenClock.Installer.exe)
```

Comment:
Well, i planned on have this one because if you see a repo with 3k downloads (because it is on the update process) it looses reputatuon, and counting that people are not likely to install third-party apps, they wwill be less likely if it has 2k~5k downloads, because it looks kinda sus or less legit

Reply:
Ok, changed to "Total" downloads

![](https://img.shields.io/github/downloads/martinet101/ElevenClock/total?style=for-the-badge)

Reply:
Yes, looks like a good idea